lib/utils.ts

```diff
@@ -1,6 +1,37 @@
-import { type ClassValue, clsx } from "clsx";
-import { twMerge } from "tailwind-merge";
+import { extendedTheme } from "@/tailwind.config"
+import { type ClassValue, clsx } from "clsx"
+import { extendTailwindMerge } from "tailwind-merge"
+
+function formatColors() {
```

Comment:
I suggest: `formatTailwindColors` so we know the context/scope of the function.
And I suggest that both functions in this file goes into a file called:  `lib/helpers/tailwind.ts`